README.md

```diff
@@ -1 +1,50 @@
-# hello-world
\ No newline at end of file
+
+# Project Name: "Hello Microverse"
+
+This project is mainly about setting up diffrent development tools and working on gitHub workflows. 
+
+
+## Built With
+
+- Major languages: HTML,CSS
+- Technologies used: linters, Lighthouse, Webhint, Stylelint
+
+
+## Getting Started
+
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+- web browser 
+### Usage
+- clone this repository into your local machine
+- unzip it
+- open the 'index.html' file with your browser
+
+## Authors
+
+👤 **Mulugeta M.**
+
+- GitHub: [@githubhandle](https://github.com/belmeetmule)
+- Twitter: [@twitterhandle](https://twitter.com/belmeetmule)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/belmeetmule)
+
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/belmeetmule/hello-world/issues).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- To team microverse and those who wrote the readme template
+
+## 📝 License
+
+This project is [MIT](./MIT.md) licensed.
```

Comment:
- [ ] Kindly make sure to fix this broken link to make your documentation more professional. You can do this by adding the `[MIT.md](https://github.com/microverseinc/readme-template/blob/master/MIT.md)` file in the root directory. You can edit it according to your information like the name,...